docs/conf.py

```diff
@@ -56,7 +56,7 @@ def get_version():
 intersphinx_mapping = {
     'python': ('https://python.readthedocs.io/en/latest/', None),
     'django': ('https://django.readthedocs.io/en/1.11.x/', None),
```

Comment:
We should probably link to the canonical docs of these other two, also.

Reply:
I'll open another PR for this. This is approved so I'm going to merge it.

Reply:
https://github.com/readthedocs/readthedocs.org/pull/6085